cmd/daemon/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Printf("WICD placeholder...")
```

Comment:
While this is fine to do for a start, we need to get the logging story right for WICD the 2nd time around. So please think through what logging mechanism will be used. 

@sebsoto if you have some particular logger in mind, I suggest to start using that from the beginning.

Reply:
I would suggest we use klog in order to maintain uniformity with the other on-node programs

Reply:
@sebsoto should I start using that in this PR?

Reply:
Personally I don't think that's necessary as the ticket only requires a no-op WICD and nothing else

Reply:
because this is simple PR, I suggest adding some basic logs like "process running or something" ( in that way we know log files getting created and stored in the right place). A simple one to knock it along with this PR.

Reply:
Added